backend/apps/matching/views/participant_info_edit.py

```diff
@@ -18,6 +19,11 @@ class ParticipantInfoUpdateView(UpdateView):
     slug_url_kwarg = "uuid"
     slug_field = "uuid"
 
+    def dispatch(self, *args, **kwargs):
+        if self.request.user.participant().info.uuid not in self.request.path:
```

Comment:
This is not the nicest solution, but I could not think of a better one that works without a db-lookup. The url contains the uuid of the `participant_info`, not the participant or the user.

Reply:
I would do it in reverse, as described in my comment below